src/pages/Books.js

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import Book from '../components/Book';
 import Form from '../components/Form';
 
-const Home = () => (
+const Books = () => (
   <>
-    <Book title=" Oliver Twist" author="Charles Dickens" />
+    <Book title="Dracula" author="Arthur Conan Doyle" />
     <Form />
```

Comment:
- [ ] I am adding this review here because you did not change the Form component in this project. I am unable to add any books. Your app basically does nothing. Kindly refactor the Form component to add book details using the local state and `onChange` event listeners.

  This book should be dispatched and stored in the store. You can visit [this article](https://www.newline.co/@andreeamaco/how-to-create-a-react-form-controlled-vs-uncontrolled-components--3eb7a468) for a pointer